packages/arb-token-bridge-ui/package.json

```diff
@@ -42,12 +42,14 @@
     "react-virtualized": "^9.22.3",
     "swr": "^2.1.2",
     "tailwind-merge": "^0.9.0",
+    "ts-node": "^10.9.1",
     "use-query-params": "^2.1.1",
-    "wagmi": "^0.12.18"
+    "wagmi": "^0.12.18",
+    "zustand": "^4.3.8"
   },
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "yarn workspace arb-token-bridge-ui generateDenylist && next build",
```

Comment:
we could create a `postbuild` script that will automatically execute after build: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

also unsure if we need to list out the yarn workspace here as we're running the script inside of it? will just `yarn  generateDenylist` work?

Reply:
done, but it has to be `prebuild` so I made it such